metadata-ingestion/README.md

```diff
@@ -355,6 +356,24 @@ source:
     # table_pattern/schema_pattern is same as above
 ```
 
+### AWS Glue `glue`
+Extracts:
+- List of tables
+- Column types associated with each table
+- Table metadata, such as owner, description and parameters
+```yml
+source:
+  type: glue
+  config:
+    aws_region: aws_region_name # i.e. "eu-west-1"
+    env: environment used for the DatasetSnapshot URN, one of "DEV", "EI", "PROD" or "CORP". # Optional, defaults to "PROD".
+    databases: list of databases to process. # Optional, if not specified then all databases will be processed.
```

Comment:
would like to standardize on the allow-deny-list pattern for these across all the sources.

e.g. schema_pattern and table_pattern in SQL-based sources such as 
https://datahubproject.io/docs/metadata-ingestion#mysql-metadata-mysql
 
Not a show-stopper, and can be done in a follow-on PR.